gradle/formatting.gradle

```diff
@@ -163,7 +163,7 @@ subprojects {
         java {
           // Normally this isn't necessary, but we have Java sources in
           // non-standard places
-          target '**/*.java'
+          target 'src/**/*.java'
```

Comment:
narrow down the tasks inputs here to avoid implicit dependencies on others tasks output we do not want here (and is deprecated with gradle 7.0)